front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -54,9 +49,7 @@ export const useStoreDataForSpeedLimitByTagSelector = ({
     }
   }, []);
 
-  const speedLimitsByTags = isStdcm
-    ? COMPOSITION_CODES
-    : compact(uniq(concat(speedLimitTags, speedLimitsTagsByInfraId)));
+  const speedLimitsByTags = isStdcm ? COMPOSITION_CODES : compact(uniq(speedLimitsTagsByInfraId));
```

Comment:
We don't need to `compact` anymore, we can only have `string` returned by the endpoint.
```suggestion
  const speedLimitsByTags = isStdcm ? COMPOSITION_CODES : uniq(speedLimitsTagsByInfraId);
```

Or the other files you changed, we can also remove `compact` but to do so you'll have to do the same logic as here when you call the endpoint : `data: speedLimitsTagsByInfraId = []`